src/Microsoft.DotNet.Build.CloudTestTasks/AzureBlobLease.cs

```diff
@@ -134,7 +134,7 @@ private static void AutoRenewLeaseOnBlob(AzureBlobLease instance, string account
                     using (HttpClient client = new HttpClient())
                     {
                         Tuple<string, string> leaseAction = new Tuple<string, string>("x-ms-lease-action", "renew");
-                        Tuple<string, string> headerLeaseId = new Tuple<string, string>("x-ms-lease-id", leaseId);
+                        Tuple<string, string> headerLeaseId = new Tuple<string, string>("x-ms-lease-id", instance._leaseId);
```

Comment:
What's the purpose of all the variables passed to the method, if they are just being acquired from the insance of AzureBlobLease?  ie, are accountName, accountKey, containerName, etc... needed or could they be replaced with instance._accountName, instance._blah?

Reply:
I think we wanted to be able to call this function outside of autorenew as well.

Reply:
Makes sense

Reply:
Actually there isn't anyone consuming this except us, I went ahead and removed the extra parameters. The renewal should be automatic anyways.

Reply:
I see you changed it.  Given that it's a private method, I think you're right, it's ok to change it.  Just make sure you do plenty of validation.  It'd be nice to get this in tonight, but I'd rather not rush it if we have any concerns that it could actually make publishing worse.